 #ADAPTIVE-FILTER Parameters
 adaptive_filter:

  # Filter settings
  enableImu: true
  enableWheel: true
  enableLidar: true
  filterFreq: "l"
  
  # Covariance gains
  lidarG: 10000
  wheelG: 0.05
  imuG: 0.1

